Header header
TriggerField[] fields
TriggerIngredient[] ingredients
